controller/proxy-injector/webhook.go

```diff
@@ -150,3 +158,28 @@ func (w *Webhook) inject(request *admissionv1beta1.AdmissionRequest) (*admission
 
 	return admissionResponse, nil
 }
+
+func (w *Webhook) getLabelForParent(conf *inject.ResourceConfig) (string, string, error) {
```

Comment:
I feel this belongs in the `case *v1.Pod` of the `inject.Parse()` method. All the other workloads have this label set there.

Reply:
True. That would require adding the 'k8s.API` into `inject.go`'s `ResourceConfig`, which is probably fine. What do you think @siggy and/or @klingerf ?

Reply:
As a general rule, none of the package in `./pkg/...` should depend on any of the packages in `./controller/...`. So I'd prefer not to add a new field in `ResourceConfig` that's of type `k8s.API`. That said, we could create a new field that holds a func that produces the label, and pass the func in when setting up the config.... or something along those lines.

Reply:
Sorry, I was hoping that the `AdmissionRequest` would have given us the info we need, without needing to use `k8s.API`. If this is too much work, it's probably not worth it.